routes/api.php

```diff
@@ -670,6 +670,13 @@
             ]
         );
 
+        Route::get('{user}/license_seats',
+            [
+                'as' => 'api.users.licenseseatslist',
+                'uses' => 'UsersController@license_seats'
```

Comment:
Can you switch this to the camel-cased `licenseSeats` just to keep it more consistent with the other methods?